lib/msal-node/src/client/ManagedIdentityApplication.ts

```diff
@@ -46,6 +50,7 @@ export class ManagedIdentityApplication {
     private static nodeStorage: NodeStorage | undefined;
     private networkClient: INetworkModule;
     private cryptoProvider: CryptoProvider;
+    private retryTracker: RetryTracker;
```

Comment:
Does it need to be a member of this class? It looks to me that it's more of a method variable?

Reply:
You're right, it doesn't need to be.